### PR TITLE
Add dependencies to make libvirt provisioning work.

### PR DESCRIPTION
**Pull request by p3ck**
Also - remove duplicate task.

### CODE REVIEW

File: linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml [hunk 1]
```diff
@@ -46,13 +46,6 @@
   become: yes
   register: lismt_local
 
-- name: "determine image_src mime_type"
```
[the remark this answers is not in the document]
Comment (p3ck) [reply linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml]: Look at the diff closer...  It's the exact same task twice but one of them has a comment.

`git diff -U10 HEAD^ -- linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml
diff --git a/linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml b/linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml
index 6bc0fe4..496f651 100644
--- a/linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml
+++ b/linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml
@@ -39,27 +39,20 @@
   ignore_errors: yes
 
 - name: "determine image_src mime_type"
   # identify file type
   stat:
     path: "{{ local_image_src }}"
     mime: true
   become: yes
   register: lismt_local
 
-- name: "determine image_src mime_type"
-  stat:
-    path: "{{ local_image_src }}"
-    mime: true
-  become: yes
-  register: lismt_local
-
 - name: "set image_src for uncompressed files"
   set_fact:
     img_src: "{{ local_image_src }}"
`

[the remark this answers is not in the document]
Comment (p3ck) [reply linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml]: why is github commenting so broken?  I selected code block!/??

You'll have to run the git diff command above to see what I mean...

[the remark this answers is not in the document]
Comment (herlo) [reply linchpin/provision/roles/libvirt/tasks/copy_image_src_local.yml]: Github commenting isn't broken, but the helper stuff is annoying. To make code you do three backticks followed by your code, then three more backticks. Took me a bit to get that right. I fixed your comment above, if you want to look at it you can edit it and see.

I see why I missed the diff, it was the delete and I wasn't realizing the fact that you removed it made only one. I'm good with this PR.

**Comment by p3ck** [comment]
the editor window has a bunch of buttons and one of them is "insert code", that uses single quotes.  

Thanks for merging

**Comment by herlo** [comment]
@p3ck that 'insert code' is exactly what I was talking about. It's single backticks, using 3 backticks is better. :+1:  Have a great holiday!

**Pull request merged**